src/System.IO.Pipelines/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.2.1</AssemblyVersion>
-    <PackageVersion>4.7.1</PackageVersion>
+    <PackageVersion>4.7.2</PackageVersion>
```

Comment:
For some packages u are updating minor version and for some packages you are incrementing the servicing version ? is that intentional ?

Reply:
Oh great catch, you are right, it should always be the servicing version only. Let me fix that!